src/engine/Unit_engine.ml

```diff
@@ -349,7 +349,9 @@ let regression_tests_for_lang ~polyglot_pattern_path ~with_caching files lang =
                    match_pattern ~lang
                      ~hook:(fun { Pattern_match.range_loc; _ } ->
                        let start_loc, _end_loc = range_loc in
-                       E.error "test pattern" start_loc "" Out.SemgrepMatchFound)
+                       E.error
+                         (Rule.ID.of_string "test pattern")
```

Comment:
actually space are not allowed in rule ids. Unicode chars have also been recently banned by Jonas.

Reply:
I asked @jbergler about the spec since I'm not sure where to look. I might leave a TODO for now and merge.